x-pack/plugin/security/qa/operator-privileges-tests/src/javaRestTest/java/org/elasticsearch/xpack/security/operator/Constants.java

```diff
@@ -495,6 +495,7 @@ public class Constants {
         "indices:admin/cache/clear",
         "indices:admin/data_stream/lazy_rollover",
         "indices:admin/data_stream/reindex",
+        "indices:admin/data_stream/reindex_status",
```

Comment:
Should this be `indices:admin/migration/reindex_status` to match the action name? 

More generally, do we want to change the existing related transport actions to `migration/...`? Though I'm this may not even be possible without a bunch of careful version juggling?

Reply:
Yeah good point. I _think_ it's fine to do at this point because (1) the code is inaccessible in serverless and (2) it wasn't in 8.17 so it has not made it into any release yet.